server/models/migrations/v0.8.0-v0.9.0/migrate.sql

```diff
@@ -576,4 +576,52 @@ DELIMITER ;
 -- Added the credit_balance and debit_balance property 
 -- to account for certain account references that have a debit or credit balance
 ALTER TABLE `account_reference_item` ADD COLUMN `credit_balance` TINYINT(1) NOT NULL DEFAULT 0;
-ALTER TABLE `account_reference_item` ADD COLUMN `debit_balance` TINYINT(1) NOT NULL DEFAULT 0;
\ No newline at end of file
+ALTER TABLE `account_reference_item` ADD COLUMN `debit_balance` TINYINT(1) NOT NULL DEFAULT 0;
+
+
+
+
+
+
+
+
+INSERT INTO unit VALUES
+(213, '[OHADA] Compte de resultat','TREE.OHADA_RESULT_ACCOUNT','',144,'/modules/reports/ohada_profit_loss','/reports/ohada_profit_loss');
+
+INSERT INTO `report` (`id`, `report_key`, `title_key`) VALUES
+(26, 'ohada_profit_loss', 'TREE.OHADA_RESULT_ACCOUNT');
+
+INSERT INTO `account_reference` VALUES (1,'RA','Achat de marchandises',NULL,0),
+(2,'RB','Variation de stocks',NULL,0),
+(3,'RC','Achat de matières premières et fournitures',NULL,0),
+(4,'RD','Variation de stocks',NULL,0),
+(5,'RE','Autres achats',NULL,0),
+(6,'RH','Variation de stoks',NULL,0),
+(7,'RI','Transports',NULL,0),
+(8,'RJ','Services extérieurs',NULL,0),
+(9,'RK','Impôt et taxes',NULL,0),
+(10,'RL','Autres charges',NULL,0),
+(11,'RP','Charges de personnel',NULL,0),
+(12,'RS','Dotations aux amortissements et aux provisions',NULL,0),
+(13,'SA','Frais financiers',NULL,0),
+(14,'SC','Pertes et change',NULL,0),
+(15,'SD','Dotations aux amortissements et aux provisions',NULL,0),
+(16,'SK','Valeurs comptables de cessions',NULL,0),
+(17,'SL','Charges HAO',NULL,0),(18,'SM','Dotations HAO',NULL,0),
+(19,'SR','Impoôt sur le résultat',NULL,0),
+(20,'TA','Ventes de marchandises',NULL,0),
+(21,'TD','Travaux, service vendus',NULL,0),
+(22,'TE','Production stockée',NULL,0),
+(23,'TF','Production immobilisée',NULL,0),
+(24,'TL','Autres produits',NULL,0),
+(25,'TS','Reprises de provisions',NULL,0),
+(26,'TT','Transport de charges',NULL,0),
+(27,'UA','Revenus financiers',NULL,0),
+(28,'UC','Gains de change',NULL,0),
+(29,'UD','Reprises de provision',NULL,0),
+(30,'UE','Transferts de charges',NULL,0),
+(31,'UK',"Produits de cessions d'immobilisations",NULL,0),
+(32,'UL','Produits HAO',NULL,0),(33,'UM','Reprises HAO',NULL,0),(34,'UN','Transferts de charges',NULL,0);
+
+INSERT INTO `account_reference_item` VALUES (1,1,2958,0),
+(2,2,2973,0),(3,3,2967,0),(4,4,2984,0),(5,5,2996,0),(6,5,3012,0),(7,6,2987,0),(8,7,3036,0),(9,8,3058,0),(10,8,3100,0),(11,9,3132,0),(12,10,3158,0),(13,11,3178,0),(14,12,3234,0),(15,12,3260,0),(16,13,3218,0),(17,13,3227,1),(18,14,3227,0),(19,15,3277,0),(20,16,3411,0),(21,17,3419,0),(22,18,3431,0),(23,19,3447,0),(24,20,3282,0),(25,21,3292,0),(26,22,3351,0),(27,23,3346,0),(28,24,3355,0),(29,25,3390,0),(30,25,3382,0),(31,26,3385,0),(32,27,3372,0),(33,27,3377,1),(34,28,3377,0),(35,29,3407,0),(36,30,3387,0),(37,31,3415,0),(38,32,3425,0),(39,32,3442,0),(40,32,3429,1),(41,33,3436,0),(42,34,3429,0);
```

Comment:
Having [account_reference](https://github.com/IMA-WorldHealth/bhima-2.X/pull/3273/files#diff-b313cc94659d42ded761e5f823018467R594) and [account_reference_item](https://github.com/IMA-WorldHealth/bhima-2.X/pull/3273/files#diff-b313cc94659d42ded761e5f823018467R626) here are not necessary, since account ids used are for IMCK. It will be better to place this code into a separate file for an easy integration to the IMCK database.